From: csagan5 <32685696+csagan5@users.noreply.github.com>
Date: Fri, 31 Jul 2020 08:27:26 +0200
Subject: Enable prefetch-privacy-changes by default

Prevents the browser from initiating undesired network connections & prefetching.

Initial Commit: https://github.com/uazo/cromite/commit/f513eb76ecb9c5ae62e065608596ccb13643b1c6#diff-0e7340cb0ba472cc6d8b6d93414b4ea9ba91d0f4e9489b12125368822676db33 (Inherited from Bromite)
Category: Privacy
License: GPL-3.0-only - https://spdx.org/licenses/GPL-3.0-only.html
---
 .../features_cc/Enable-prefetch-privacy-changes-by-default.inc   | 1 +
 1 file changed, 1 insertion(+)
 create mode 100644 cromite_flags/third_party/blink/common/features_cc/Enable-prefetch-privacy-changes-by-default.inc

diff --git a/cromite_flags/third_party/blink/common/features_cc/Enable-prefetch-privacy-changes-by-default.inc b/cromite_flags/third_party/blink/common/features_cc/Enable-prefetch-privacy-changes-by-default.inc
new file mode 100644
--- /dev/null
+++ b/cromite_flags/third_party/blink/common/features_cc/Enable-prefetch-privacy-changes-by-default.inc
@@ -0,0 +1 @@
+SET_CROMITE_FEATURE_ENABLED(kPrefetchPrivacyChanges);
--
